### ISSUE TITLE
Font sizes?

### ISSUE THREAD

**Issue by borgauf** [emacs-dashboard/emacs-dashboard#120]
The font sizes of my actual "recent files" on my dashboard are rather small. How can I increase this? I'm guessing it's not a dashboard issue?

**Comment by jcs090218** [comment]
Yeah, I don't think this is the `dashboard`'s issue. Can you post a screenshot so the team can confirm this issue easier?

Thank you!
